### PR TITLE
Technology history activation and sea raiding nerf

### PR DESCRIPTION
**Pull request by rquinio**
Changes to prevent tribal pagans sea raiding frenzy, by:
- lowering shipbuilding technology below 1 (technology history)
- removing shipyards (province history) 

It lowers the technological level in 470 for all, compared to previous vanilla defaults.
As this has lots of impact, and may break game balance, I'm doing a PR for review and discussion !

### CODE REVIEW

**Comment by loup99** [comment]
Could the Vandals & Wisigoths keep the ships? Same for everybody in Britain?

**Comment by rquinio** [comment]
I'm pretty sure Vandals and Visigoths have some ships at start, as there are in some western technological group.
Feudal in Britain have ships but not the Celtic and Germanic tribal pagans, otherwise they would indeed raid the Mediterranean coasts...

File: WTWSMS/history/technology/western_europe.txt [hunk 1]
```diff
@@ -67,13 +97,10 @@ technology = {
 		d_epidaurum 
 		d_sicily 
 		d_picenum_suburbicarium 
-		d_euregius 
 		d_bodensee 
-		d_tripolitania 
 		d_numidia 
 		d_caesariensis 
 		d_carthago
```
Comment (rquinio) [inline WTWSMS/history/technology/western_europe.txt]: Here are the ships for Vandals.

**Comment by loup99** [comment]
In that case, I see no problems with pushing this to main.

**Comment by loup99** [comment]
I will push this as I see no problems with it.

**Pull request merged**